java/maven/src/org/netbeans/modules/maven/customizer/CustomizerProviderImpl.java

```diff
@@ -415,10 +414,6 @@ private static void writeNbActionsModel(final Project project, final FileObject
             @Override
             public void run() throws IOException {
                 JDOMFactory factory = new DefaultJDOMFactory();
-                
-                InputStream inStr = null;
-                FileLock lock = null;
-                OutputStreamWriter outStr = null;
                 try {
                     Document doc;
                     if (mapping.getActions().isEmpty()) { //#224450 don't write empty nbactions.xml files
```

Comment:
Please retain the `FileLock` for fo, as it prevents concurrent overwrite of `fo` contents after for is read and before a modified version is written.

Reply:
I put the `FileLock` inside a `twr` in the line 438 of the new version.